lib/ansible/plugins/action/uri.py

```diff
@@ -58,6 +60,7 @@ def run(self, tmp=None, task_vars=None):
                     raise AnsibleActionFail(
                         'body must be mapping, cannot be type %s' % body.__class__.__name__
                     )
+                new_body = deepcopy(body)
                 for field, value in body.items():
```

Comment:
```suggestion
                for field, value in new_body.items():
```

I was a bit confused by the change on 78/81, so I think we should instead just loop the new body, and remove the change on 78/81.